io/module_write_netcdf.F90

```diff
@@ -200,11 +200,11 @@ subroutine write_netcdf(wrtfb, filename, &
 
        if (par) then
           ncerr = nf90_create(trim(filename),&
-                  cmode=IOR(IOR(NF90_CLOBBER,NF90_NETCDF4),NF90_CLASSIC_MODEL),&
+                  cmode=IOR(NF90_CLOBBER,NF90_NETCDF4),&
```

Comment:
Does this mean we will not output classic netcdf files?

Reply:
Yes, this will create netcdf4 file.